tests/unit_node/http2_test.ts

```diff
@@ -165,3 +166,23 @@ Deno.test("[node/http2 client GET https://www.example.com]", async () => {
   assertEquals(status, 200);
   assert(chunk.length > 0);
 });
+
+Deno.test("[node/http2.createServer()]", { sanitizeOps: false }, async () => {
```

Comment:
The tests are failing when we have two http2.createServer() in the same file. I'm not sure how the tests are processed by Deno.test underneath but I'll explore what's the cause.

Reply:
The previous test was timing out because we no longer emit "stream" on a session in the current implement. I replaced the test with the new test instead.